lib/api/readable.js

```diff
@@ -267,29 +267,53 @@ function consumeStart (consume) {
   }
 }
 
+const decoder = new TextDecoder()
+
+/**
+ * @see https://encoding.spec.whatwg.org/#utf-8-decode
+ * @param {Buffer} buffer
+ */
+function utf8DecodeBytes (buffer) {
```

Comment:
import it from the fetch utils instead of duplicating

Reply:
I just changed the implementation and it is ok.

Reply:
not really, the code is duplicated when it doesn't need to be

Reply:
Sorry for the misunderstanding. 
Just wanted to say that this is not a problem since we have changed the implementation.

Reply:
@KhafraDev 
is this all right with you?https://github.com/nodejs/undici/pull/2510/commits/8b07631f4626e46451d23d8bf1bfbeca4648a492